modules/administration/pages/live-patching-channel-setup.adoc

```diff
@@ -0,0 +1,78 @@
+[[live-patching-channel-setup]]
+= Set up Channels for Live Patching
+
+A reboot is required every time you update the full kernel package.
+Therefore, it is important that clients using live patching do not have newer kernels available in the channels they are assigned to.
```

Comment:
See later... or capitalize here (and later on where I did not add suggestions, though):
```suggestion
Therefore, it is important that clients using Live Patching do not have newer kernels available in the channels they are assigned to.
```